refactor_me.py

```diff
@@ -3,15 +3,16 @@
 from collections import namedtuple, defaultdict
 from operator import itemgetter
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+
+Expense = namedtuple('Expense', ('type', 'amount'))
```

Comment:
renaming `type` to `category` would be a better change

Reply:
true, even better. But while I was out of the context, I've just cast a casual look on it...